libc/cmake/modules/LLVMLibCHeaderRules.cmake

```diff
@@ -97,8 +94,11 @@ function(add_gen_header target_name)
   set(absolute_path ${CMAKE_CURRENT_SOURCE_DIR}/${ADD_GEN_HDR_GEN_HDR})
   file(RELATIVE_PATH relative_path ${LIBC_INCLUDE_SOURCE_DIR} ${absolute_path})
   set(out_file ${LIBC_INCLUDE_DIR}/${relative_path})
+  set(dep_file "${out_file}.d")
+  file(RELATIVE_PATH rel_out_file ${CMAKE_BINARY_DIR} ${out_file})
```

Comment:
CMake uses absolute paths everywhere so there's no point in relativizing these.

Reply:
Done.